tests/unit/base/SGObject_unittest.cc

```diff
@@ -311,3 +312,87 @@ TEST(SGObject,parameter_hash_changed)
 	SG_UNREF(inf);
 }
 #endif //USE_GPL_SHOGUN
+
+TEST(SGObject, tags_set_float)
+{
+	Tag<float64_t> width("width");
+	CGaussianKernel* gauss_kernel = new CGaussianKernel();
+	EXPECT_EQ(gauss_kernel->set(width, 10.0), 10.0);
```

Comment:
not sure I get the reasoning here: why should set return things that were there previously?